experiments/openebs/openebs-pool-disk-loss/openebs_pool_disk_loss_ansible_logic.yml

```diff
@@ -126,7 +135,17 @@
       - name: Injecting the chaos
         include_tasks: "{{ c_util }}"
 
-      # POST CHAOS CHECKS
+      # POST CHAOS 
+      
+      ## RECORD EVENT FOR POST-CHAOS CHECK
+      - include_tasks: /utils/common/generate-kubernetes-chaos-events.yml
+        vars:
+          stage: "PostChaosCheck"
+          exp_pod_name: "{{ chaos_pod_name }}"
+          engine_ns: "{{ a_ns }}"
+          message: "AUT is Running successfully"
```

Comment:
Can you please add this task just after checking the running status of AUT and auxiliary apps?